components/markets/MarketScroll.tsx

```diff
@@ -81,9 +81,11 @@ const MarketScroll = ({
   return (
     <div
       ref={containerRef}
-      className="grid grid-cols-1 gap-y-7 sm:grid-cols-2 sm:gap-7 lg:grid-cols-3"
+      className="grid grid-cols-1 gap-y-2 sm:grid-cols-2 sm:gap-7 lg:grid-cols-3"
```

Comment:
the `go to markets` button and buttons are off here it seems

![Screenshot 2024-02-29 at 1 50 06 PM](https://github.com/zeitgeistpm/ui/assets/14626415/7fe1cff0-3c49-432f-8f97-c74718583dde)